activesupport/lib/active_support/cache/strategy/local_cache.rb

```diff
@@ -48,8 +48,8 @@ def clear(options = nil)
             @data.clear
           end
 
-          def fetch(*args, &block)
-            @data.fetch(*args, &block)
+          def fetch_entry(key, options=nil) # :nodoc:
```

Comment:
Ah, nevermind, just saw you were the one who added `fetch`. Though let's move `fetch_entry` to a private section:

```ruby
private
  def fetch_entry(*args, options = nil)
    # ...
  end
```